lib/loader/index.js

```diff
@@ -135,6 +186,16 @@ module.exports = {
       var firstScript = getFirstScript();
 
       firstScript.parentNode.insertBefore(script, firstScript);
+
+      _loadedUrls[url] = true;
+
+      if (options.namespaceName) {
+        namespace.loadedUrls = _loadedUrls;
```

Comment:
@sirugh, @reason-bv Just to be clear, you guys are talking about something like...
```js
_loadedUrls[url] = true;

if (options.namespaceName) {
  if (namespace.loadedUrls) {
    namespace.loadedUrls = _loadedUrls;
  }
  else {
    namespace.registerProperty('loadedUrls', _loadedUrls);
  }
}
else {
  loadedUrls = _loadedUrls;
}
```